package.json

```diff
@@ -4,30 +4,38 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "nyc --reporter=html --reporter=text mocha --require @babel/register --require babel-polyfill ./test/ --recursive --exit"
+    "test": "npm run test:migrate && NODE_ENV=test nyc --reporter=html --reporter=text mocha --require @babel/register --require babel-polyfill ./test/ --recursive --exit",
+    "migrate": "sequelize db:migrate",
+    "undoAll:migrate": "sequelize db:migrate:undo:all",
+    "undoCurrent:migrate": "sequelize db:migrate:undo",
```

Comment:
Please use `"undo-all:migrate"` and "undo-current:migrate" here instead. @chekwas88